src/hub.html

```diff
@@ -335,6 +338,19 @@
                 </a-entity>
             </template>
 
+            <template id="camera-hover-menu">
+                <a-entity class="ui interactable-ui hover-container camera-snap-menu" mixin="button-container">
+                    <a-entity mixin="rounded-text-action-button" snap-camera-button position="0 -0.145 0.0" scale="0.75 0.75 0.75">
+                        <a-entity text="value:; width:1; align:center;" text-raycast-hack position="0 0 0.02"></a-entity>
+                        <a-image
+                            icon-button="image: #snap-camera; hoverImage: #snap-camera;"
```

Comment:
You created a hover image but didn't add it to assets or use it here.

Reply:
ah that image is dead code, will remove